cmake/scripts/common/AddonHelpers.cmake

```diff
@@ -120,16 +120,18 @@ macro (build_addon target prefix libs)
             foreach(src_file ${USED_SOURCES})
               file(STRINGS ${src_file} BIN_ADDON_SRC_PARTS)
               foreach(loop_var ${BIN_ADDON_SRC_PARTS})
+                string(REGEX REPLACE ".*#*.*include|.*#*.*import" "#include" loop_var "${loop_var}")
```

Comment:
This is wrong as it removes all characters before `#` and between `#` and `include` or `import`, instead of removing only white space characters.